src/chrome/content/rules/ArchiveTeam.org.xml

```diff
@@ -0,0 +1,27 @@
+<!--
+        Invalid certificate:
+                tracker.archiveteam.org:1337
+
+        Redirect to HTTP:
+
+
+        Refused:
+                fileformats.archiveteam.org
+                digitize.archiveteam.org
+                iabak.archiveteam.org
+                justsolve.archiveteam.org
+
+
+        Time out:
+                metadata.archiveteam.org
+
+-->
+<ruleset name="ArchiveTeam.org">
+    <target host="archiveteam.org/"/>
+    <target host="www.archiveteam.org/"/>
+    <target host="tracker.archiveteam.org/"/>
+    <target host="warriorhq.archiveteam.org/"/>
+    <securecookie host=".+" name=".+"/>
+
```

Comment:
`tracker.archiveteam.org:1337` redirects to `tracker.archiveteam.org:1338` so you should add the additional rule:

```
	<rule from="^http://tracker\.archiveteam\.org(:1337)?/" to="https://tracker.archiveteam.org:1338/"/>
```

though I'm not sure this is the correct form :P